ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -563,6 +563,75 @@ def test_torch_nll_loss(
     )
 
 
+# gaussian_nll_loss
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.gaussian_nll_loss",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=0.01,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    dtype_and_target=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=0.0,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    dtype_and_var=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        allow_inf=False,
+        min_value=0.0,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    eps=st.floats(
+        min_value=0.0,
+        max_value=1.0,
+        allow_nan=False,
+        allow_infinity=False,
+    ),
+    reduction=st.sampled_from(["mean", "sum"]),
```

Comment:
I didn't notice this before, but you're not testing for different `full` values. You need to generate and pass True and False values for the argument `full`.

Reply:
Added them.